packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/AddBankYapily/Authorize/template.success.tsx

```diff
@@ -0,0 +1,218 @@
+import React, { useState } from 'react'
+import { FormattedMessage } from 'react-intl'
+import styled from 'styled-components'
+
+import { Button, Icon, Image, Text } from 'blockchain-info-components'
+import { FlyoutWrapper, Row, Title } from 'components/Flyout'
+import { AddBankStepType } from 'data/types'
+
+import { Props as _P } from '.'
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+`
+const BackContainer = styled(Text)`
+  display: flex;
+  align-items: center;
+  width: 100%;
+  font-weight: 600;
+  font-size: 20px;
+`
+const DropdownTitleRow = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  cursor: pointer;
+  /* chevorn icon rotation */
+  > span:last-child {
+    size: 10px;
+    transition: transform 0.2s;
+    color: ${props => props.theme.grey600};
+    &.active {
+      transform: rotate(180deg);
+    }
+  }
+`
+const InfoTitle = styled(Title)`
+  font-weight: 600px;
+  line-height: 1.5;
+  color: ${props => props.theme.grey900};
+`
+
+const InfoDropdown = styled.div`
+  max-height: 0;
+  margin-top: 0;
+  overflow: hidden;
+  transition: max-height, margin-top 0.3s;
+  &.isToggled {
+    max-height: 100%;
+    margin-top: 12px;
+  }
+`
+const InfoText = styled(Title)`
+  font-size: 14px;
+  font-weight: 500;
+  color: ${props => props.theme.grey600};
+  line-height: 1.5;
+`
+const Bottom = styled(FlyoutWrapper)`
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  height: 100%;
+`
+const DropdownItem = ({ bodyText, titleText }) => {
+  const [isToggled, handleToggle] = useState(false)
+  return (
+    <Row>
+      <DropdownTitleRow onClick={() => handleToggle(!isToggled)}>
+        <InfoTitle>{titleText}</InfoTitle>
+        <Icon name='chevron-down' className={isToggled ? 'active' : ''} />
+      </DropdownTitleRow>
+      <InfoDropdown className={isToggled ? 'isToggled' : ''}>
+        <InfoText>{bodyText}</InfoText>
+      </InfoDropdown>
+    </Row>
+  )
+}
+
+const Success: React.FC<Props> = props => {
```

Comment:
(minor) this could be function component

Reply:
👍🏻 fixing